sdk/eventhub/Azure.Messaging.EventHubs.Processor/tests/Snippets/Sample03_EventProcessorHandlersLiveTests.cs

```diff
@@ -432,19 +449,10 @@ async Task processErrorHandler(ProcessErrorEventArgs args)
                         return;
```

Comment:
Interesting, is OOM something that we expect users to explicitly handle?

Reply:
Not as a general rule; if they hit an OOM condition, it's not likely that we're going to even be healthy enough to ping the handler.    I don't recall why this exists this way, other than to just demonstrate the types of things that can be done.   I'll take a look and see if it makes sense to keep it here.

Reply:
I don't think that it makes sense in context;  removed.